### ISSUE TITLE
How to train with the detection boxes？

### ISSUE THREAD

**Issue by kingthreestones** [Buki2/MAttNet-pyt1#1]
just like the title

**Comment by Buki2** [comment]
Thank you for your interest in our code repository.

According to the implementation details in the paper, MAttNet uses only ground-truth boxes in the model training. From the code of the loss function, it is difficult to directly replace the ground-truth boxes with detection boxes.
